### PR TITLE
SpriteTurning

### PR DESCRIPTION
**Pull request by Ashthecool**
The sprite now looks to the side it's walking. Except forward and backwards

### CODE REVIEW

File: my_game.py
Comment (tobiasbp) [inline my_game.py:220]: This will not work if the sprite uses a graphic of a different size. You should flip the graphic instead.

**Review by tobiasbp — CHANGES_REQUESTED** [review]
Using size for flip will break if the graphics are changed.

File: my_game.py
Comment (Ashthecool) [reply my_game.py:220]: That doesn't work. I tried it. But it only works when I created the object of the player. I can't seem to make it flip like that. Trust me. I tried that. A lot. Could we work on another solution?

Comment (tobiasbp) [reply my_game.py:220]: > That doesn't work. I tried it. But it only works when I created the object of the player. I can't seem to make it flip like that. Trust me. I tried that. A lot. Could we work on another solution?

Show me the code

Comment (Ashthecool) [reply my_game.py:220]: I

> > That doesn't work. I tried it. But it only works when I created the object of the player. I can't seem to make it flip like that. Trust me. I tried that. A lot. Could we work on another solution?
> 
> Show me the code

It's not that. It's just that it doesn't do anything. We're insensibly making a new variable for the self.player. It doesn't know what "flipped horizontally" is. It's not a thing you can call and make it a bool.

Comment (tobiasbp) [reply my_game.py:220]: You are right... I see it now. You have to load the graphics twice, and flip one. Then switch between them. This should happen in the _Player_ class. You need to implement the switching between textures by adding the method _update()_ like you can see in the following example:

https://api.arcade.academy/en/latest/examples/sprite_face_left_or_right.html#sprite-face-left-or-right

Comment (tobiasbp) [reply my_game.py:220]: Actually, I think we need to move the knowledge of what keys move the player(s) into the player class. Then we can flip the graphic when we change direction from within the player object...

Comment (Ashthecool) [reply my_game.py:220]: Deal, I'll wait until you merged yours into it. And then I'll do that. 👍

**Pull request closed without merging**
